build/pipelines/azure-pipelines.ci-internal.yaml

```diff
@@ -20,16 +20,24 @@ jobs:
 - template: ./templates/build-app-internal.yaml
   parameters:
     platform: x86
+    condition: not(eq(variables['Build.Reason'], 'PullRequest'))
 
 - template: ./templates/build-app-internal.yaml
   parameters:
     platform: ARM
+    condition: not(eq(variables['Build.Reason'], 'PullRequest'))
 
 - template: ./templates/run-ui-tests.yaml
   parameters:
     platform: x64
     runsettingsFileName: CalculatorUITests.release.runsettings
 
+- template: ./templates/run-ui-tests.yaml
+  parameters:
+    platform: x86
+    runsettingsFileName: CalculatorUITests.release.runsettings
+    condition: not(eq(variables['Build.Reason'], 'PullRequest'))
+
```

Comment:
Run all the UI tests as azure-pipelines.release.yaml previously did. After ci-internal passes the build and UI test, then trigger release pipeline, and UI test will not run in release pipeline.